EasyClangComplete.sublime-settings

```diff
@@ -28,6 +28,14 @@
   "cpp_flags" : [
     "-std=c++11"
   ],
+  // Objective-C specific flags. Prepend common_flags for Objective-C files.
+  "objective_c_flags" : [
+    "-std=c11"
+  ],
+  // Objective-C++ specific flags. Prepend common_flags for Objective-C++ files.
+  "objective_cpp_flags" : [
+    "-std=c++11"
```

Comment:
I am a bit confused. You removed the same line in `.clang_complete` file. If it is ok, why did you need to remove it?